src/mame/heathkit/tlb.cpp

```diff
@@ -1122,6 +1134,25 @@ ioport_constructor heath_superset_tlb_device::device_input_ports() const
 	return INPUT_PORTS_NAME(superset);
 }
 
+uint8_t heath_superset_tlb_device::crtc_reg_r(offs_t reg)
+{
+	m_reverse_video_disabled = bool(BIT(reg, 3));
+
+	return heath_tlb_device::crtc_reg_r(reg);
+}
```

Comment:
Updated, but I'm a little confused about this. I also updated the other read that had side effects. But if this prevents the code from running, the system is not going to be accurate.

Reply:
The idea is that you don't want anything that changes the machine state to happen due to the debugger's reads rather than the CPU's.  For instance, on systems with read-to-trigger I/O locations (the Apple II is a classic example) I don't want to have opening a debugger memory view window change the video mode or click the speaker, and the side_effets_disabled() test prevents that while still allowing the system to run normally.